src/metpy/calc/indices.py

```diff
@@ -42,6 +42,12 @@ def precipitable_water(dewpt, pressure, bottom=None, top=None):
     `pint.Quantity`
         The precipitable water in the layer
 
+    Examples
+    --------
+    >>> pressure = np.array([1000, 950, 900]) * units.hPa
+    >>> dewpoint = np.array([20,15,10]) * units.degC
```

Comment:
I guess we’re not flake8-ing the docstrings. Pedantic, but should be spaces after the commas.

Reply:
Good catch - I agree that it helps with reading comprehension.